src/KeyboardLayout_sv_SE.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Swedish keyboard layout.
+ */
+
+#include "KeyboardLayout.h"
+
+extern const uint8_t KeyboardLayout_sv_SE[128] PROGMEM =
+{
+	0x00,          // NUL
+	0x00,          // SOH
+	0x00,          // STX
+	0x00,          // ETX
+	0x00,          // EOT
+	0x00,          // ENQ
+	0x00,          // ACK
+	0x00,          // BEL
+	0x2a,          // BS  Backspace
+	0x2b,          // TAB Tab
+	0x28,          // LF  Enter
+	0x00,          // VT
+	0x00,          // FF
+	0x00,          // CR
+	0x00,          // SO
+	0x00,          // SI
+	0x00,          // DEL
+	0x00,          // DC1
+	0x00,          // DC2
+	0x00,          // DC3
+	0x00,          // DC4
+	0x00,          // NAK
+	0x00,          // SYN
+	0x00,          // ETB
+	0x00,          // CAN
+	0x00,          // EM
+	0x00,          // SUB
+	0x00,          // ESC
+	0x00,          // FS
+	0x00,          // GS
+	0x00,          // RS
+	0x00,          // US
+
+	0x2c,          // ' '
+	0x1e|SHIFT,    // !
+	0x1f|SHIFT,    // "
+	0x20|SHIFT,    // #
+	0x21|SHIFT,    // $
+	0x22|SHIFT,    // %
+	0x23|SHIFT,    // &
+	0x31,          // '
+	0x25|SHIFT,    // (
+	0x26|SHIFT,    // )
+	0x32|SHIFT,    // *
+	0x2d,          // +
+	0x36,          // ,
+	0x38,          // -
+	0x37,          // .
+	0x24|SHIFT,    // /
+	0x27,          // 0
+	0x1e,          // 1
+	0x1f,          // 2
+	0x20,          // 3
+	0x21,          // 4
+	0x22,          // 5
+	0x23,          // 6
+	0x24,          // 7
+	0x25,          // 8
+	0x26,          // 9
+	0x37|SHIFT,    // :
+	0x36|SHIFT,    // ;
+	0x64,          // <
```

Comment:
Hmm. hmm. I had problems with this character, and I am sure I specifically tested it because this key was very relevant to the project I'm working on. I will confirm.

Reply:
I guess it should work as long as it is not used with <kbd>Shift</kbd>.

Reply:
This message typed on a keyboard interfaced through the keyboard library using 0x64 for <

< works.
shift + < gives > (correct)
altgr + < gives \ (correct)

There is a fourth symbol on the keycap... no idea what combination of modifiers is required... resembles |... but otherwise I'm confident the keyboard scancode for < is correct.

Reply:
Confirmed correct comb for <